chemprop/data/data.py

```diff
@@ -400,7 +400,7 @@ def gt_targets(self) -> List[np.ndarray]:
         """
 
         """
```

Comment:
Oops left out the description
```suggestion
        """
        Returns indications of whether the targets associated with each molecule are greater-than inequalities.
        
        :return: A list of lists of booleans indicating whether the targets in those positions are greater-than inequality targets.
        """
```